src/components/App/index.tsx

```diff
@@ -1,38 +1,21 @@
-import React, { memo, FC, Component, PureComponent, createElement } from 'react';
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
 
-import styles from './styles.module.scss';
-
-function FunctionComponentExample() {
-  return <h2>Function Component Example</h2>;
-}
-
-class ClassComponentExample extends Component {
-  render() {
-    return <h2>Class Component Example</h2>;
-  }
-}
-
-class PureComponentExample extends PureComponent {
-  render() {
-    return <h2>Pure Component Example</h2>;
-  }
-}
+import ErrorBoundary from 'components/_common/ErrorBoundary';
 
-const CreateElementExample = () => {
-  return createElement('h2', null, 'Create Element Example');
-};
+import MainRouter from 'routing';
+import styles from './styles.module.scss';
 
-const App: FC = () => {
+const App: React.FC = () => {
```

Comment:
Please, use destructuring here. Check all occurrences

Reply:
usually, I try to refer to this GitHub page with rules for ts+react projects - https://github.com/typescript-cheatsheets/react
![image](https://user-images.githubusercontent.com/37964216/123546067-cf9f7480-d763-11eb-9258-4e95f1d2dc77.png)


and another point, that when you test your app with ReactSpy you will need to use React.(any hook) to test hook and it a little bit confusing sometimes use React.useState and in the same file only FC<>, that's why I try to make a common view for those imports.